Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -265,11 +259,18 @@ def test_alerts_in_sentinel(self):
     # 3. Install playbooks
     # 4. Wait for the incidents to come in
     # 5. Run all these tests
-    for test in unittest.defaultTestLoader.getTestCaseNames(TestCohesity):
-        if test not in (
-            "test_cohesity_close_helios_incident",
-            "test_alerts_in_sentinel",
-        ):
-            suite.addTest(TestCohesity(test))
+    suite = unittest.TestSuite(
+        [
+            TestCohesity(test)
+            for test in unittest.defaultTestLoader.getTestCaseNames(
+                TestCohesity
+            )
+            if test
+            not in (
+                "test_cohesity_close_helios_incident",
+                "test_cohesity_close_helios_incident",
```

Comment:
Both the lines have test_cohesity_close_helios_incident.

Reply:
done